server/src/test/java/org/elasticsearch/index/mapper/MapperValidatorTests.java

```diff
@@ -21,56 +21,60 @@
 import org.elasticsearch.Version;
 import org.elasticsearch.cluster.metadata.IndexMetadata;
 import org.elasticsearch.common.Explicit;
+import org.elasticsearch.common.lucene.Lucene;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.test.ESTestCase;
 
 import java.util.Arrays;
 import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.List;
 
 import static java.util.Collections.emptyList;
 import static java.util.Collections.emptyMap;
 import static java.util.Collections.singletonList;
 
-public class MapperMergeValidatorTests extends ESTestCase {
+public class MapperValidatorTests extends ESTestCase {
```

Comment:
Do you think these should be on tests for the mappers you are validating?

Reply:
They're all for aliases, I'll rename the class to `FieldAliasMapperValidationTests` - I think it's probably still worth having as a separate class.